app/controllers/dashboard_controller.rb

```diff
@@ -35,6 +36,13 @@ def get_data
     logger.debug("DEBUG: #{current_user.inspect}")
   end
 
+  def remove_col_id
```

Comment:
Can you move the definition `remove_col_id` to `ApplicationController` and remove some of the repeated code?